catalog/designers/floriankarsten/bio.html

```diff
@@ -0,0 +1,2 @@
+<p>Florian Karsten Studio (Brno, Czech Republic) focuses on graphic design, type design and programming. We create websites, books, programmes, typefaces and above all, functional systems. We’re excited about open-source and peer2peer networks.</p>
```

Comment:
I suggest not using "we" but "they"

Reply:
Yes, something sounded a bit strange in that "we". Changed